src/createNativeStackNavigator.tsx

```diff
@@ -44,6 +44,7 @@ import {
   StackNavigationProp,
   Layout,
 } from 'react-navigation-stack/src/vendor/types';
+import { isSearchBarAvailableForCurrentPlatform } from './searchBarPlatforms';
```

Comment:
I'd move this to `./utils/searchBarPlatforms`, and then export it in `index.native.tsx` and `index.tsx`, so v5 `native-stack` and this file can import it straight from `react-native-screens`, same as other things (see lines 10-22).